src/lib.rs

```diff
@@ -49,10 +49,13 @@ mod error;
 mod iter;
 pub mod parse;
 
-/// Reexports of extension traits.
-pub mod exts {
-    pub use super::display::DisplayHex;
-    pub use super::parse::FromHex;
+/// Reexports of the common crate types and traits.
+pub mod prelude {
+    #[doc(inline)]
+    pub use crate::{
+        display::DisplayHex,
+        parse::{FromHex, HexToArrayError, HexToBytesError},
```

Comment:
Do we really need the errors in prelude? I think most users will not use them unless they are making a library in which case they probably shouldn't use prelude at all.

Reply:
Ah yes, I agree. Will remove.

Reply:
Removeing the error types makes this PR a straight re-name of `exts` to `prelude`. I re-wrote the git log to say as much but left the PR descriptions as it was.